lib/styles/components/badge.scss

```diff
@@ -0,0 +1,26 @@
+.nui-badge {
```

Comment:
@yedhink These styles are specific for `MenuBar` component. It should not be applied globally.